subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_coap

```diff
@@ -36,6 +36,13 @@ config NRF_CLOUD_COAP_GF_CONF
 	  configuration values do_reply, fallback, and hi_conf. When the cloud
 	  support is ready, NRF_CLOUD_COAP_GF_CONF can be set true.
 
+config NRF_CLOUD_COAP_KEEPOPEN
```

Comment:
No, it is not available on the NRF9160.

Reply:
Ah, duh, I misread that if statement.

ANyways the root of my concern is what happens on Wi-Fi CoAP builds. It would be nice if there were a clean way to enforce that this flag is not enabled unless we are specifically using hardware that supports the CID/SO_KEEPOPEN feature

Or does zephyr native support these features?

Reply:
Good point.

Reply:
No, looks like SO_KEEPOPEN is an NCS-specific cellular socket option.

I'm adding a qualifier on SOC type.